learn-regular-expressions-by-building-a-spam-filter/spamFilter080.js

```diff
@@ -0,0 +1,29 @@
+const isSpam = msg => {
+  const blacklistRegexes = [
+    /v[i1|]agra/i,
+    /free money/i,
+    /work from home/i,
+    /stock alert/i,
+    /dear fr[i1|]end/i
+  ];
+  /*
+  Other characters can be mangled too. Luckily, we've managed to obtain a copy of _The Spammer's Handbook_ by S. Pam Merchant, which helpfully provides a lookup table of all the characters they use:
+
+  Character | Mangled versions
```

Comment:
The table here is really cool, but I think it would be better to have users create character classes one by one for each character. "i" is taken care of, so you could work through the rest of the list starting from "a".